code_generator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -108,7 +108,10 @@ namespace <#= qualifiedNamespace #>
                 Unity.Entities.Entity entity;
                 if (!view.TryGetEntity(op.EntityId.Id, out entity))
                 {
-                    Debug.LogErrorFormat(TranslationErrors.OpReceivedButNoEntity, op.GetType().Name, op.EntityId.Id);
+                    view.LogDispatcher.HandleLog(new LogEvent("Entity not found.", LogType.Error)
+                        .WithField("Id", op.EntityId.Id)
+                        .WithField("IdType", IdType.EntityId)
```

Comment:
This doesn't let you put both an ecs and a spatail ID in one message. Might be better to key the id on the id type.